aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -450,7 +450,9 @@ The `Protocols` property establishes the HTTP protocols (`HttpProtocols`) enable
 |--|--|
 | `Http1` | HTTP/1.1 only. Can be used with or without TLS. |
 | `Http2` | HTTP/2 only. May be used without TLS only if the client supports a [Prior Knowledge mode](https://tools.ietf.org/html/rfc7540#section-3.4). |
+| `Http3` | HTTP/3 only. Requires TLS. The client may need to be configured to use HTTP/3 only. |
 | `Http1AndHttp2` | HTTP/1.1 and HTTP/2. HTTP/2 requires the client to select HTTP/2 in the TLS [Application-Layer Protocol Negotiation (ALPN)](https://tools.ietf.org/html/rfc7301#section-3) handshake; otherwise, the connection defaults to HTTP/1.1. |
+| `Http1AndHttp2AndHttp3` | HTTP/1.1, HTTP/2 and HTTP/3. The first client request normally uses HTTP/1.1 or HTTP/2, and the [`alt-svc` response header prompts the client to upgrade to HTTP/3](xref:fundamentals/servers/kestrel/http3#alt-svc). HTTP/2 and HTTP/3 requires TLS; otherwise, the connection defaults to HTTP/1.1. |
```

Comment:
That's a long link 😆 
```suggestion
| `Http1AndHttp2AndHttp3` | HTTP/1.1, HTTP/2 and HTTP/3. The first client request normally uses HTTP/1.1 or HTTP/2, and the [`alt-svc` response header](xref:fundamentals/servers/kestrel/http3#alt-svc) prompts the client to upgrade to HTTP/3. HTTP/2 and HTTP/3 requires TLS; otherwise, the connection defaults to HTTP/1.1. |
```